shared/js/background/chrome-events.es6.js

```diff
@@ -56,10 +57,27 @@ chrome.webRequest.onHeadersReceived.addListener(
             // returns a promise
             return ATB.updateSetAtb(request)
         }
+
+        // Strip 3rd party response header
+        const tab = tabManager.get({ tabId: request.tabId })
+        if (!request.responseHeaders) return
+        if (tab && tab.site.whitelisted) return
+        if (tab && utils.isFirstParty(request.url, tab.url)) return
+        const index = request.responseHeaders.findIndex(header => { return header.name.toLowerCase() === 'set-cookie' })
+        if (index !== -1) {
+            if (!cookieConfig.isExcluded(request.url)) {
+                request.responseHeaders.splice(index, 1)
+            }
+        }
+
+        return { responseHeaders: request.responseHeaders }
     },
     {
         urls: ['<all_urls>']
-    }
+    },
+    [
+        'blocking', 'responseHeaders', 'extraHeaders'
```

Comment:
Firefox doesn't support the `extraHeaders` option here which causes an exception when the extension is started (`Uncaught Error: Type error for parameter extraInfoSpec (Error processing 2: Invalid enumeration value "extraHeaders") for webRequest.onHeadersReceived.`)

You can feature detect support for these flags via `chrome.webRequest.OnHeadersReceivedOptions`. e.g.
```js
const extraInfoSpec = ['blocking', 'responseHeaders'];
if (chrome.webRequest.OnHeadersReceivedOptions.EXTRA_HEADERS) {
   extraInfoSpec.push(chrome.webRequest.OnHeadersReceivedOptions.EXTRA_HEADERS)
}
```